ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -4,13 +4,15 @@
 
 import frappe
 from erpnext.accounts.doctype.sales_invoice.sales_invoice import SalesInvoice
+from erpnext.setup.doctype.branch.branch import Branch
+from frappe import _
```

Comment:
Use `ft` from `translation.py` instead. IDEs/static analyzers treat `_` as a protected member since it starts with an underscore and typically issue a warning.